README.md

```diff
@@ -1,8 +1,60 @@
+
 # flask-starter
 
+  
+
+  
+
 ## Starting the server:
 
+  
+
+  
 
 1. Open a terminal and go to the server folder. Make sure you have **pipenv** installed (`pip install pipenv`)
+
+  
+
 2. Install the dependencies with `pipenv install`. This also createa a virtual environment, if there isn't one already
+
+  
+
 3. Activate the virtual environment and start the app with `pipenv run flask run`
+
+  
+
+  
+
+## Setting up PostgreSQL
+
+  
+
+1. Follow the instructions to [install](https://www.postgresqltutorial.com/install-postgresql/) Postgresql (v12) on your local machine 
+
+2. Install all dependencies with `pipenv install`
+
+3. (MacOS and Windows only) To run the psql shell from anywhere in our command line, we need to set up our $PATH [environment variable](https://superuser.com/questions/284342/what-are-path-and-other-environment-variables-and-how-can-i-set-or-use-them) to point towards the location of our psql shell executable.
+
+	- Windows: [follow these instructions](https://sqlbackupandftp.com/blog/setting-windows-path-for-postgres-tools)
+	- [MacOS](https://opensource.com/article/17/6/set-path-linux): `export PATH=/Library/PostgreSQL/12/bin:$PATH`
```

Comment:
Didn't catch this before - this links to a Linux article for Mac

Reply:
oops, I've already merged it to the dev branch. What do you think is the best way to go about this? Should I create another pull request?

Reply:
No worries, approval is supposed to mean it's good to merge! It's not time sensitive, so you can just add it to your next PR, whatever task that happens to be

Reply:
sounds good!